drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -409,11 +409,19 @@ function DragAndDropTemplates(configuration) {
                     return h("li", {innerHTML: message.message});
                 }))
             ];
-            popup_content = h("div.popup-content", {}, have_messages ? content_items : []);
+            popup_content = h(
+                                ctx.last_action_correct ? "div.popup-content" : "div.popup-content.popup-content-incorrect",
+                                {},
+                                have_messages ? content_items : []
+                            );
```

Comment:
Please change indentation of these lines, so that they are indented like this:

```javascript
            popup_content = h(
                ctx.last_action_correct ? "div.popup-content" : "div.popup-content.popup-content-incorrect",
                {},
                have_messages ? content_items : []
            );
```